sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -380,7 +381,12 @@ def visit_transform(self, transform_node):
         # pylint: disable=wrong-import-order, wrong-import-position
         from apache_beam import Flatten
         if isinstance(transform_node.transform, Flatten):
-          output_pcoll = transform_node.outputs[None]
+          from apache_beam.runners.portability.fn_api_runner.translations \
+            import only_element
+          output_pcoll = (
+              transform_node.outputs[None] if None in transform_node.outputs
+              else only_element(transform_node.outputs.values()))
```

Comment:
`only_element` is fine to use everywhere, no need to branch on the key being None.